src/execution_plan/optimizations/reduce_count.c

```diff
@@ -8,7 +8,7 @@
 #include "../ops/ops.h"
 #include "../../util/arr.h"
 
-static GrB_Monoid edgeCountMonoid = NULL;
+static GrB_UnaryOp countMultipleEdges = NULL;
```

Comment:
I think it's acceptable to not free this, but we should add a comment to that effect.